### ISSUE TITLE
evote.python.org

### ISSUE THREAD

**Issue by JulienPalard** [python/psf-salt#152]
Here https://raw.githubusercontent.com/python/psf-salt/master/docs/list.rst there's an evote.python.org machine documented, but it does not resolve to an IP:

```$ dig evote.python.org

; <<>> DiG 9.11.5-1-Debian <<>> evote.python.org
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 31990
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;evote.python.org.		IN	A

;; Query time: 1 msec
;; SERVER: 10.0.0.2#53(10.0.0.2)
;; WHEN: Wed Nov 07 23:29:59 CET 2018
;; MSG SIZE  rcvd: 45
```

has it been partially removed in https://github.com/python/psf-salt/commit/378249e92e707a101dc091cc012c574f3cdb5969?

**Comment by ewdurbin** [comment]
Indeed, that server has been decommissioned, as we're using helios for PSF elections moving forward. I'll update the list.

**Comment by ewdurbin** [comment]
resolved in 90603286d825d0e5a6f3fb4475cc9420fab940b8

**Comment by JulienPalard** [comment]
:+1:
